### PR TITLE
Use guzzle http client

### PR DESCRIPTION
**Pull request by ryanannatuvasquez**
## Description
- Use https://github.com/guzzle/guzzle as our http client library. 
- Yay no more SVN library dependencies 😅

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -122,7 +132,7 @@ public static function delete($url, array $params = array())
      */
     public static function getCode()
     {
-        return (self::$http instanceof HTTPclient === true) ? self::$http->lastcode() : null;
+        return (self::$response instanceof Response) ? self::$response->getStatusCode() : null;
```
Comment (markjezreeldagot) [inline src/Client.php]: does it need to have parenthesis?
return self::$response instanceof Response ? self::$response->getStatusCode() : null;
short hands for if else doesn't require parenthesis unless there are operations needed

File: src/Client.php [hunk 2]
```diff
@@ -134,7 +144,7 @@ public static function getCode()
      */
     public static function getHeaders()
     {
-        return (self::$http instanceof HTTPclient === true) ? self::$http->lastheaders() : null;
+        return (self::$response instanceof Response) ? self::$response->getHeaders() : null;
```
Comment (markjezreeldagot) [inline src/Client.php]: same with the above comment

**Comment by ryanannatuvasquez** [comment]
@eshleebien @markjezreeldagot PTAL

File: src/Client.php [hunk 3]
```diff
@@ -181,10 +185,10 @@ private static function _request($url, $method, array $params, array $files = []
                     'timeout' => Config::get('timeout'),
                     'allow_redirects' => ['max' => 1],
                 ],
-                'base_url' => Config::get('baseurl'),
             ]);
         }
 
+        $url = Config::get('baseurl').$url;
```
Comment (eshleebien) [inline src/Client.php]: (nit) space before and after ```.```

File: src/Client.php [hunk 4]
```diff
@@ -134,7 +146,9 @@ public static function getCode()
      */
     public static function getHeaders()
     {
-        return (self::$http instanceof HTTPclient === true) ? self::$http->lastheaders() : null;
+        return self::$response instanceof Response
```
Comment (markjezreeldagot) [inline src/Client.php]: I think it should be no new lines just single line.
return self::$response instanceof Response ? self::$response->getHeaders() : null;

File: src/Client.php [hunk 5]
```diff
@@ -122,7 +132,9 @@ public static function delete($url, array $params = array())
      */
     public static function getCode()
     {
-        return (self::$http instanceof HTTPclient === true) ? self::$http->lastcode() : null;
+        return self::$response instanceof Response
```
Comment (markjezreeldagot) [inline src/Client.php]: I think it should be no new lines just single line.
return self::$response instanceof Response ? self::$response->getHeaders() : null;

Comment (ryanannatuvasquez) [reply src/Client.php]: good point. however its too long for a short hand if. basically in PSR2 only wants limit to 80 per line. Thanks

File: src/Client.php (hunk 4 above)
Comment (ryanannatuvasquez) [reply src/Client.php]: please read comment below

**Pull request merged**